### PR TITLE
feat(filters): add chrome-extension:// in browser extensions filtering

### PR DESCRIPTION
**Pull request by gierschv**

### CODE REVIEW

File: src/sentry/filters/browser_extensions.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
     r'static\.woopra\.com\/js\/woopra\.js',
     # Chrome extensions
     r'^chrome:\/\/',
+    r'^chrome-extension:\/\/',
```
Comment (mattrobenolt) [inline src/sentry/filters/browser_extensions.py]: Can we just merge this with the line above as:

```python
r'^chrome(?:-extension)?:\/\/',
```

**Comment by gierschv** [comment]
@mattrobenolt Ok, updated

**Review by mattrobenolt — APPROVED** [review]
:pizza: 🍦

**Comment by mattrobenolt** [comment]
Thanks!

**Pull request merged**